libs/template/file.go

```diff
@@ -68,13 +77,24 @@ func (f *copyFile) PersistToDisk() error {
 		return err
 	}
 	defer srcFile.Close()
-	dstFile, err := os.OpenFile(path, os.O_CREATE|os.O_EXCL|os.O_WRONLY, f.perm)
-	if err != nil {
+
+	ctx := context.Background()
+
+	if runsOnDatabricks(ctx) && f.isNotebook() {
```

Comment:
I'd recommend adding the `HasPrefix("/Workspace")` check here as well. That way we use the same `os.Write` call for all files instead of two with different configurations.

We could also get rid of the `writeNotebook` function and inline the import API call.

Reply:
done